ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthSendRawTransaction.java

```diff
@@ -64,12 +68,14 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
 
     final Transaction transaction;
     try {
-      transaction = DomainObjectDecodeUtils.decodeRawTransaction(rawTransaction);
-    } catch (final InvalidJsonRpcRequestException e) {
+      transaction = Transaction.readFrom(RLP.input(Bytes.fromHexString(rawTransaction)));
```

Comment:
Did we determine that the rpc format is expecting RLP encoded txData rather than transaction byte prefixed opaque data?   https://github.com/hyperledger/besu/pull/2100#issuecomment-812051091

Reply:
It works with both since`Transaction.readFrom` calls `TransactionDecorder.decodeForWire` which has the logic to decode both mode

Reply:
It works with both since `Transaction.readFrom` calls `TransactionDecorder.decodeForWire` which has the logic to decode both mode